src/public/stageView.ts

```diff
@@ -0,0 +1,82 @@
+import { sendMessageToParent } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { SdkError } from '../public';
+import { FrameContexts } from './constants';
+
+/**
+ * Namespace to interact with the stage view specific part of the SDK.
+ */
+export namespace stageView {
+  /**
+   * Parameters to open a stage view.
+   */
+  export interface StageViewParams {
+    /**
+     * The application ID of the Teams application to be opened.
+     * Required
+     */
+    appId: string;
+
+    /**
+     * The URL of the content to display.
+     * Required
+     */
+    contentUrl: string;
+
+    /**
+     * The Teams app website URL.
+     * Not required but still used to populate an open website button in stage view.
+     */
+    websiteUrl?: string;
+
+    /**
+     * The name of the stage view.
+     * Delete seems unused
+     */
+    name?: string;
+
+    /**
+     * The entity ID.
+     * Not sure
+     */
+    entityId?: string;
+
+    /**
+     * The chat or channel ID.
+     * Not sure
+     */
+    threadId?: string;
+
+    /**
+     * The initatiator of the stage view request.
+     * Delete seems unused
+     */
+    source?: string;
+
+    /**
+     * The title to give the stage view.
+     * Used by the error dialog.  Should this be required?
```

Comment:
What is the error dialog you're referring to here? Can we just use the `name` in that as well? Can you please share some examples of real world values for the `name` and `title`? When are these different?